sourced/ml/transformers/repo2df.py

```diff
@@ -1,33 +1,30 @@
 from sourced.ml.transformers import Transformer
 
 
-class Repo2DocFreq(Transformer):
-    NDOCS_KEY = -1, 0
+class Documents2DocFreq(Transformer):
 
-    def __init__(self, extractors, **kwargs):
+    def __init__(self, extractors, document_column='blob_id', **kwargs):
```

Comment:
We should not give this (or any) default

Reply:
And according to the default value, the name is bad. Should be `id_column`

Reply:
default just for back compatibility. will be removed. Just need to make changes in Apollo also.

`id_column ` also bad. I can read it as `identifiers column`. can rename to `document_id`?

Reply:
We can but this file has nothing to do with identifiers.
`document_id` does not imply that we expect a column name.
We can use `key_column`